pkg/routeagent-driver/cni-driver/cni-iface.go

```diff
@@ -0,0 +1,67 @@
+package cni_driver
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net"
+
+	"k8s.io/klog"
+
+	"github.com/submariner-io/admiral/pkg/log"
+)
+
+type CniInterface struct {
+	Name      string
+	IPAddress string
+}
+
+func DiscoverCNIInterface(clusterCIDR string) (*CniInterface, error) {
```

Comment:
Discover  (avoid shuttering)

Reply:
Hehe I was proposing to rename the module to cni_interface.

So it would be cni_interface.Discover()